### PR TITLE
Unifying indentation: even-more-python-for-beginners-... chapter 11-15

### LINKED ISSUE #45: Unifying indentation: even-more-python-for-beginners-... chapter 11-15
According to #20, I made sub-issue of it.
Unifying indentation to `4 spaces indent` according to convention of #20, dealing with even-more-python-for-beginners-... chapter 11-15

### PR DESCRIPTION
**Pull request by devquint**
Resolves #45

### CODE REVIEW

File: even-more-python-for-beginners-data-tools/15 - Visualizing data with Matplotlib/15 - Visualizing correlations.ipynb [hunk 1]
```diff
@@ -43,9 +39,7 @@
   },
   {
    "cell_type": "markdown",
-   "execution_count": null,
```
Comment (Hyuntae-Jeong) [inline even-more-python-for-beginners-data-tools/15 - Visualizing data with Matplotlib/15 - Visualizing correlations.ipynb:46]: @devquint Why was this line deleted?

Comment (devquint) [reply even-more-python-for-beginners-data-tools/15 - Visualizing data with Matplotlib/15 - Visualizing correlations.ipynb:46]: There might be some environment-related things on jupyter notebook.

**Review by Hyuntae-Jeong — APPROVED** [review]

**Review by jesusleejy — APPROVED** [review]
